manifests/params.pp

```diff
@@ -1,11 +1,24 @@
 # PHP params class
 #
+# === Parameters
+#
+# [*config_root_override*]
+#   The configuration root directory used in parameter calculation
+#
+# [*php_version_override*]
+#   The version of php used in parameter calculation
+#
 class php::params(
-  $cfg_root = undef, # lint:ignore:parameter_documentation
+  $config_root_override = undef,
+  $php_version_override = undef
```

Comment:
I'm not sure if we should add even more parameters to this class. The params class is inherited by the main `php` class and thus these parameters can't be overridden without Hiera. Quite a few of our users demanded that we make the module independent of Hiera. What do you think?